src/components/manage/Blocks/Title/Edit.jsx

```diff
@@ -143,13 +143,33 @@ export const TitleBlockEdit = (props) => {
     onSelectBlock(block);
   }, [block, onSelectBlock]);
 
-  const renderElement = useCallback(({ attributes, children }) => {
-    return (
-      <h1 {...attributes} className="documentFirstHeading">
-        {children}
-      </h1>
-    );
-  }, []);
+  const renderElement = useCallback(
+    ({ attributes, children }) => {
+      const hasType = properties['@type'];
+      const AboveTitle =
+        hasType &&
+        config.getComponent({ name: 'AboveTitle', dependencies: [hasType] })
+          .component;
+
+      return (
+        <>
+          {AboveTitle ? (
+            <>
+              <AboveTitle item={properties} />
+              <h1 {...attributes} className="documentFirstHeading">
+                {children}
+              </h1>
+            </>
+          ) : (
+            <h1 {...attributes} className="documentFirstHeading">
+              {children}
+            </h1>
+          )}
```

Comment:
Is there any reason not to do it like this?

```suggestion
          <>
            {AboveTitle ? <AboveTitle item={properties} /> : null}
            <h1 {...attributes} className="documentFirstHeading">
              {children}
            </h1>
          </>
```

Reply:
no reason, it looks a lot better now, thanks!